script.js

```diff
@@ -1,4 +1,68 @@
 var main = function (input) {
   var myOutputValue = 'hello world';
-  return myOutputValue;
-};
+  var words = computerGenResult();
```

Comment:
Please start function names with verbs for clarity that they are functions. See here for naming conventions for different variable types. https://bootcamp.rocketacademy.co/course-logistics/naming-casing-and-commenting-conventions